docs/selective_attention_network.md

```diff
@@ -0,0 +1,104 @@
+
+<p align="center">
+  <img src="../img/console/AI Models/Training-Config-Upload.png">
+</p>
+
+These are the settings you can modify:
+
+* __data -> input_dim__: input_dim is the input resolution to the network to be trained. The input_dim must be a multiple of 32. Minimum value is 64 and maximum value is 1024. For e.g. 64, 96, 128, 160, ..., 1024.
+
+!!! Guide
+    * Use 64 if your objects of interest are large sizes (e.g. faces up to 1 m away or vehicles up to 5 m away).
+    * Use 128 if your objects of interest are small to medium sizes (e.g. faces up to 3 m away or vehicles up to 20 m away).
+    * Use 256 if your objects of interest are tiny sizes (e.g. faces up to 10 m away or vehicles up to 200 m away).
+
+* __train -> batch_size__: The batch size is a number of samples processed before the model is updated. The size of a batch must be more than or equal to one and less than or equal to the number of samples in the training dataset.
+
+* __train -> epochs__: The number of epochs is the number of complete passes through the training dataset.  The number of epochs can be set to an integer value between one and infinity.
+
+* __train -> augment_multiplier__: You can use this setting to augment your training data. 1 means add 1x to the dataset (double it). 2 means add 2x (triple it).
+
+Some other settings:
+
+* __train -> max_negative_sampling_ratio__: Ratio of negative example sampling. Ratio = negative examples / positive examples
+You can keep all the other setting to default.
+
+* __test -> split_ratio__: This is a ratio for splitting a held-out test set from your dataset. The held-out test set will not be used for training. It will only be used when evaluating the model once training is completed. If value is 0.1 = 10% of dataset is split into held-out test set, and the model will be trained using remaining 90% of the data.
+
+* __transforms__: Transform settings are used to define specific transformations for data augmentation. If the value for __augment_multiplier__ is greater than 0, you can modify the __transforms__ settings depending on how you want to augment your training dataset.
+
+    To apply __Resize__, __RandomRotate__ and __RandomCrop__ transforms, you also need to set __p__ value to greater than 1.
+p = probability between 0 and 1. If __p__ is 0 then that transform is not applied. If __p__ is 0.5 then half the original images get transformed. If it's 1 then all the original images get transformed.
+
+Here is the default configuration for training a Selective Attention Network. 
+
+
+
+``` yml
+data:
+  input_dim: 256
+
+train:
+  batch_size: 64
+  epochs: 100
+  max_negative_sampling_ratio: 0.3  # Ratio of negative example sampling. Ratio = negative examples / positive examples
+  augment_multiplier: 0  # 1 means add 1x to the dataset (double it). 2 means add 2x (triple it).
+
+test:
+  split_ratio: 0.1
+
+transforms:
+  Resize:
```

Comment:
I included you on [a PR from Yousef](https://github.com/xailient/console_ml_code/pull/41) that will clean some of this up. So once that PR is merged you can update these docs based on that.

Reply:
Updated